### PR TITLE
Enable Paypal checkout

### PR DESCRIPTION
**Pull request by AlexeiaW**
#101

### CODE REVIEW

File: server/src/Packages/Paypal/Resources/js/templates/stripeFormTemplate.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const stripeFormTemplate = message => `
+<div id="stripeFormTemplate"  class="uk-container uk-container-xsmall uk-margin-large-top">
```
Comment (luke-siedle) [inline server/src/Packages/Paypal/Resources/js/templates/stripeFormTemplate.js:2]: @AlexeiaW I see there's reference to Stripe here, maybe not needed or just needs to be renamed?

File: server/src/Packages/Paypal/Resources/views/payment.blade.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
+@extends('Paypal\Payment::app')
+
+@section('content')
+<div class="uk-container uk-container-xsmall uk-margin-large-top">
+    <h1 class="uk-heading-small uk-padding-small">Complete purchase</h1>
+</div>
+<div id="stripeFormTemplate" class="uk-container uk-container-xsmall uk-margin-large-top">
```
Comment (luke-siedle) [inline server/src/Packages/Paypal/Resources/views/payment.blade.php:7]: @AlexeiaW also reference here. Looks like you do need this div, so you can just change the naming to `paypalFormTemplate`

File: server/src/Packages/Paypal/Services/PayPalClient.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    /**
+     * Set up and return PayPal PHP SDK environment with PayPal access credentials.
+     * This sample uses SandboxEnvironment. In production, use LiveEnvironment.
+     */
+    public static function environment()
+    {
+        $clientId = config("api.paypal_client_id") ?: "";
+        $clientSecret = config("api.paypal_client_secret") ?: "";
+        return new SandboxEnvironment($clientId, $clientSecret);
```
Comment (luke-siedle) [inline server/src/Packages/Paypal/Services/PayPalClient.php:32]: @AlexeiaW for switching between environments, should we make this an env or config variable?

File: server/src/Packages/Paypal/Resources/js/templates/stripeFormTemplate.js (hunk 1 above)
Comment (AlexeiaW) [reply server/src/Packages/Paypal/Resources/js/templates/stripeFormTemplate.js:2]: I removed this file

File: server/src/Packages/Paypal/Resources/views/payment.blade.php (hunk 2 above)
Comment (AlexeiaW) [reply server/src/Packages/Paypal/Resources/views/payment.blade.php:7]: renamed thank you

File: server/src/Packages/Paypal/Services/PayPalClient.php (hunk 3 above)
Comment (AlexeiaW) [reply server/src/Packages/Paypal/Services/PayPalClient.php]: good idea, thanks for that, I implemented it

**Pull request merged**